test/ApiFactoryTest.php

```diff
@@ -148,7 +148,7 @@ public function testCreateApi()
 
         $this->assertEquals('Test', $api->getName());
         $this->assertEquals(1, $api->getVersion());
-        $this->assertCount(4, $api->getServices());
+        $this->assertCount(5, $api->getServices());
```

Comment:
i thought it was best to include a new service and new test instead of updating the existing rest service, test, and assertions. unfortunately that means i had to increment here. please let me know if you'd prefer that i do something different.